ReleaseProcessAutomation.IntegrationTests/ReleaseProcessScript.Test.Config

```diff
@@ -2,7 +2,7 @@
 <settings>
     <jira>
         <jiraUrl>https://re-motion.atlassian.net</jiraUrl>
-        <jiraProjectKey>GHTEST</jiraProjectKey>
+        <jiraProjectKey>DEFINETLYNOTAPROJECTKEY</jiraProjectKey>
```

Comment:
Same questions as above, do we still need that?

Reply:
*makes more sense to keep the change

Reply:
If we do not have any tests that make use of it we can keep the incorrect project key, otherwise we should use a correct one to allow easier testing. You do need to authenticate before you can do anything bad so I don't think it is that bad to have a correct key in here. The `GHTEST` project is also only used for testing so there shouldn't be anything bad possible.